### PR TITLE
STENCIL-2448: Add expanded view of swatch on hover

### PR DESCRIPTION
**Pull request by bc-ong**
Add a larger view of a swatch image when option is hovered over.   

screen shot 2017-02-15 at 1 31 44 pm

@mjschock @mcampa

### CODE REVIEW

**Comment by mcampa** [comment]
Can you please add an entry to the `CHANGELOG.md` file?

**Comment by mcampa** [comment]
Also, can you post a screenshot on how this looks :)

File: assets/scss/layouts/products/_productSwatch.scss [hunk 1]
```diff
@@ -12,8 +12,44 @@ $position_of_x : str-index($value_of_swatch_size, "x");
 $first_value : str-slice($value_of_swatch_size, 0, $position_of_x - 1);
 $second_value : str-slice($value_of_swatch_size, $position_of_x + 1);
 
+[data-product-attribute] .form-option.form-option-swatch {
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSwatch.scss]: should this be `[data-product-attribute-value]` instead?

Comment (bc-ong) [inline assets/scss/layouts/products/_productSwatch.scss]: This selector is overriding the following selector: [data-product-attribute] .form-option

**Comment by shanth100** [comment]
💚

**Pull request merged**